waspc/src/Wasp.hs

```diff
@@ -31,6 +31,8 @@ module Wasp
     , module Wasp.Page
     , getPages
     , addPage
+    
+    , module Wasp.Route
```

Comment:
Aha so we are exporting whole modules like this. Although I think at some point we stopped doing this and instead now we import specifically what we need -> if it is Route, we import Wasp.Route. So maybe we can not do this? You can see above that button or actions or entity form don't have whole module exported.

Reply:
aha cool, yes I remember now. Ok then I will not export it, let's continue with that practice.